CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 
  - removed internal sd route
  - auth for self description endpoint
+ - add requestedArtifact as mandatory field for communication with UC App (UC App v1.7.0 and DataApp v0.2.7)
```

Comment:
Can we move DataApp v0.2.7 to the line removed internal sd route?
Or maybe to put note section saying that because of those 2 changes those 2 2 versions required